apps/ledger-live-desktop/src/renderer/families/tron/StepReceiveFunds.jsx

```diff
@@ -0,0 +1,308 @@
+// @flow
```

Comment:
Not mandatory, but might be cool if we can create at least new files as .tsx instead of .jsx 👌

Reply:
Very relevant, but we're in a hurry indeed...